### ISSUE TITLE
is Touch ID supported on device returning false on simulator

### ISSUE THREAD

**Issue by liltimtim** [Chris-Corea/BiometricAuthenticator#1]
function .isTouchIdEnabledOnDevice and .isTouchIdSupportedOnDevice returns false even if simulator is enrolled in Touch ID.  According to Apple Docs, if the simulator is enrolled, then it should be able to simulator having Touch ID support and also have Touch ID enabled. 

Further investigation into the code

```
    func supportsTouchId() -> Bool {
        let invalidModels = [
            "x86_64",                   // Simulator
            "iPhone4,1",                // iPhone 4S
            "iPhone5,1", "iPhone5,2",   // iPhone 5
            "iPhone5,3", "iPhone5,4",   // iPhone 5C
            "iPhone10,3", "iPhone10,6"  // iPhone X (CDMA and GSM)
        ]
        if let modelName = deviceSystemName() {
            return !invalidModels.contains(modelName)
        }
        return false
    }
```

This denotes that the simulator doesn't support Touch ID when in fact it does. 

![screen shot 2018-08-28 at 7 23 37 am](https://user-images.githubusercontent.com/4106065/44720027-4e937080-aa93-11e8-8547-44f42d04bf3d.png)

**Comment by liltimtim** [comment]
I've opened a pull request which fixes / addresses the simulator issue @Chris-Corea 

#2

**Comment by Chris-Corea** [comment]
Good catch and thanks for reporting this.

**Comment by Chris-Corea** [comment]
Merged #2. I'll publish the pod with the change ASAP.

**Comment by liltimtim** [comment]
thank you @Chris-Corea :) glad to help contribute.
